cmd/libsnap-confine-private/snap.c

```diff
@@ -43,3 +50,94 @@ bool verify_security_tag(const char *security_tag)
 
 	return (status == 0);
 }
+
+static int skip_lowercase_letters(const char **p)
+{
+	int skipped = 0;
+	for (const char *c = *p; islower(*c); ++c) {
```

Comment:
`*c >= 'a' && *c <= 'z'` instead of `islower`. We don't want any locale-dependent logic here.